samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysis_Cpp.sln

```diff
@@ -1,83 +1,107 @@
+		{C573E45A-CA10-4A97-9AAA-99A301383A68}.Release|x64.Deploy.0 = Release|x64
+		{C573E45A-CA10-4A97-9AAA-99A301383A68}.Release|x86.ActiveCfg = Release|x86
+		{C573E45A-CA10-4A97-9AAA-99A301383A68}.Release|x86.Build.0 = Release|x86
+		{C573E45A-CA10-4A97-9AAA-99A301383A68}.Release|x86.Deploy.0 = Release|x86
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Debug|ARM.ActiveCfg = Debug|ARM
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Debug|ARM.Build.0 = Debug|ARM
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Debug|x64.ActiveCfg = Debug|x64
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Debug|x64.Build.0 = Debug|x64
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Debug|x86.ActiveCfg = Debug|Win32
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Debug|x86.Build.0 = Debug|Win32
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Release|ARM.ActiveCfg = Release|ARM
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Release|ARM.Build.0 = Release|ARM
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Release|x64.ActiveCfg = Release|x64
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Release|x64.Build.0 = Release|x64
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Release|x86.ActiveCfg = Release|Win32
+		{E0320926-075F-46FC-9CC0-865B635BB1A1}.Release|x86.Build.0 = Release|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Debug|ARM.ActiveCfg = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Debug|x64.ActiveCfg = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Debug|x86.ActiveCfg = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Debug|x86.Build.0 = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Release|ARM.ActiveCfg = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Release|ARM.Build.0 = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Release|x64.ActiveCfg = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Release|x64.Build.0 = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Release|x86.ActiveCfg = Debug|Win32
+		{38FA18AA-D23B-4B38-AB5E-ED7891A6588B}.Release|x86.Build.0 = Debug|Win32
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Debug|ARM.ActiveCfg = Debug|ARM
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Debug|ARM.Build.0 = Debug|ARM
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Debug|x64.ActiveCfg = Debug|x64
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Debug|x64.Build.0 = Debug|x64
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Debug|x86.ActiveCfg = Debug|Win32
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Debug|x86.Build.0 = Debug|Win32
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Release|ARM.ActiveCfg = Release|ARM
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Release|ARM.Build.0 = Release|ARM
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Release|x64.ActiveCfg = Release|x64
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Release|x64.Build.0 = Release|x64
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Release|x86.ActiveCfg = Release|Win32
+		{8D4B9659-6AB7-4EB0-919A-517FAD43DFE2}.Release|x86.Build.0 = Release|Win32
+		{7EC87A87-91F9-4C2C-9DEA-8FE75F76F17C}.Debug|ARM.ActiveCfg = Debug|Win32

```

Comment:
>Debug|ARM.ActiveCfg = Debug|Win32 [](start = 41, length = 33)

mismatch, same goes for Release, make sure all arch are built correctly (may have to edit by hand)